src/main/resources/hudson/maven/Messages_da.properties

```diff
@@ -24,21 +24,24 @@ MavenModuleSetBuild.DiscoveredModule=Opdagede et nyt modul {0} {1}
 MavenBuild.Triggering=Starter et nyt byg af {0}
 MavenBuild.KeptBecauseOfParent=Beholdt da {0} er gemt
 MavenModuleSetBuild.FailedToParsePom=Kunne ikke parse POM''erne
-MavenBuilder.Waiting=Venter p\u00e5 at Jenkins bliver f\u00e6rdig med at samle data
+MavenBuilder.Waiting=Venter p\u00E5 at Jenkins bliver f\u00E6rdig med at samle data
```

Comment:
@recena this is unreviewable. You can use [this script](https://github.com/jenkinsci/jenkins/blob/27aeca9ee28999e5532b6e2a285e9c3306a4ac16/core/move-l10n.groovy) with modifications to the basedir for moving I18N lines without mangling unrelated parts of the text.